lib/screens/profile_screens/contribution/interactor/viewmodels/contribution_bloc.dart

```diff
@@ -39,50 +40,56 @@ class ContributionBloc extends Bloc<ContributionEvent, ContributionState> {
         emit(state.copyWith(
             pageState: PageState.success,
             pageCommand: NavigateToScoreDetails(
-              score: 0,
-              scoreType: 'Todo1',
-              title: 'Todo2',
-              subtitle: 'Todo3',
-            )));
+                score: state.score?.contributionScore?.value ?? 0,
+                scoreType: 'Contribution',
+                title: ' Your total contribution is measured by combining your other scores. ',
+                subtitle:
+                    ' It determines how much Trust points you earn for making governance decisions, and also increases how many Seeds you earn from the Harvest by measuring your participation. ')));
         break;
       case ScoreType.transactionScore:
         emit(state.copyWith(
             pageState: PageState.success,
             pageCommand: NavigateToScoreDetails(
-              score: 0,
-              scoreType: 'Todo1',
-              title: 'Todo2',
-              subtitle: 'Todo3',
+              score: state.score?.transactionScore?.value ?? 0,
+              scoreType: 'Transaction',
```

Comment:
i see `Transaction` here

Pls make scoreType an ENUM with an extension function for the to string